.github/workflows/hotfix-wf-service.yml

```
@@ -0,0 +1,400 @@
+    runs-on: ubuntu-latest
+    needs: [release,build-and-push-image]
+    env:
+      GH_TOKEN: ${{ github.token }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      # TODO: add caching for docker_full_image which build previously
+
+      - name: Run Container and Copy File
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:5:11: Double quote to prevent globbing and word splitting

(shellcheck)